sites/public/styles/overrides.scss

```diff
@@ -56,9 +56,41 @@
           border: none;
         }
 
+        .navbar-dropdown-container {
+          margin-left: -144px;
```

Comment:
Why do we need a -144px margin here? What are we counteracting?

Reply:
This was basically pulled from SiteHeader.scss. I've kind of reorganized/simplify this stuff, PTAL.